src/orion/serving/experiments_resource.py

```diff
@@ -43,9 +43,16 @@ def on_get_experiment(self, req: Request, resp: Response, name: str):
         Handle GET requests for experiments/:name where `name` is
         the user-defined name of the experiment
         """
-        try:
-            _verify_experiment_parameters(req.params)
+        error_message = _verify_experiment_parameters(req.params)
+        if error_message:
+            response = {
+                "message": error_message
+            }
+            resp.status = falcon.HTTP_400
+            resp.body = json.dumps(response)
+            return
 
+        try:
             version = int(req.params['version']) if 'version' in req.params else None
             experiment = experiment_builder.build_view(name, version)
 
```

Comment:
It may be worth moving the response creation outside of the try-except. There is nothing we want to catch there anyway, right?

Reply:
I agree. I already did this refactoring in the branch for the trial endpoint. It will be updated then.

Reply:
That's for another PR right?